packages/contracts/test/Airdrop.test.ts

```diff
@@ -299,9 +300,7 @@ describe('Airdrop', function () {
             expect(wrongAttestingFee).not.equal(attestingFee_)
```

Comment:
I don't think we need this assert because it always false. Please check it again for sure. Thanks